samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/app-in-meeting.jsx

```diff
@@ -103,20 +103,48 @@ const AppInMeeting = props => {
         });
     };
 
+    // Share the content to in view-only screen sharing mode.
```

Comment:
Hi @Prithvi-MSFT , thank you for making the updates! Just noticed I missed one small typo when suggesting the change on this line:

```suggestion
    // Share the content in view-only screen sharing mode.
```

Reply:
changed